test/services/LibraryService.test.ts

```diff
@@ -13,17 +20,32 @@ const LIBRARY_WITH_URL: fhir4.Library = {
   url: 'http://example.com'
 };
 
-const LIBRARY: fhir4.Library = {
+const LIBRARY_WITH_NO_DEPS: fhir4.Library = {
```

Comment:
This fixture seems to be exactly the same as `LIBRARY`. Is there a reason we can't just replace LIBRARY with this one now?